src/main/java/org/sciborgs1155/robot/drivetrain/DriveConstants.java

```diff
@@ -0,0 +1,211 @@
+              PROPORTIONAL_COEFFICIENT,
+              INTEGRAL_COEFFICIENT,
+              DERIVATIVE_COEFFICIENT,
+              DRIVE_CONSTRAINTS);
+      controller.setTolerance(
+          POSITION_TOLERANCE.in(Meters), VELOCITY_TOLERANCE.in(MetersPerSecond));
+      return controller;
+    }
+  }
+
+  /** Wrapper for the {@link ProfiledPIDController} constants used for rotating */
+  public static final class RotationPID {
+    public static final double PROPORTIONAL_COEFFICIENT = 1.0;
+    public static final double INTEGRAL_COEFFICIENT = 0.0;
+    public static final double DERIVATIVE_COEFFICIENT = 0.0;
+
+    /** Error tolerance */
+    public static final Measure<Angle> POSITION_TOLERANCE = Degrees.of(0.01);
+
+    /** Maximum velocity for ending rotation commands */
+    public static final Measure<Velocity<Angle>> VELOCITY_TOLERANCE = DegreesPerSecond.of(0.02);
+
+    /** Configured {@link ProfiledPIDController} */
+    public static ProfiledPIDController getController() {
+      ProfiledPIDController controller =
+          new ProfiledPIDController(
+              PROPORTIONAL_COEFFICIENT,
+              INTEGRAL_COEFFICIENT,
+              DERIVATIVE_COEFFICIENT,
+              ROTATION_CONSTRAINTS);
+      controller.setTolerance(
+          POSITION_TOLERANCE.in(Degrees), VELOCITY_TOLERANCE.in(DegreesPerSecond));
+      return controller;
+    }
+  }
+
+  /** Wrapper for the {@link SimpleMotorFeedforward} constants used for driving */
+  public static final class DriveFeedForward {
+    public static final double S = 0.0;
+    public static final double V = 0.1;
+    public static final double A = 0.01;
+
+    /**
+     * Configured {@link SimpleMotorFeedforward} controller (input MetersPerSecond, outputs voltage)
+     */
+    public static SimpleMotorFeedforward getController() {
+      return new SimpleMotorFeedforward(S, V, A);
+    }
+  }
+
+  /** Wrapper for the {@link SimpleMotorFeedforward} constants used for rotating */
+  public static final class RotationFeedForward {
+    public static final double S = 0.0;
+    public static final double V = 0.1;
+    public static final double A = 0.01;
+
+    /**
+     * Configured {@link SimpleMotorFeedforward} controller (input DegreesPerSecond, outputs
+     * voltage)
+     */
+    public static SimpleMotorFeedforward getController() {
+      return new SimpleMotorFeedforward(S, V, A);
+    }
+  }
+
+  /**
+   * If voltage magnitude is greater than maximum voltage, decreases voltage magnitude (So the
+   * motors don't explode :)
+   */
+  public static Measure<Voltage> clampVoltage(Measure<Voltage> voltage) {
+    if (Math.abs(voltage.in(Volts)) > MAX_VOLTAGE.in(Volts)) {
+      return Volts.of(Math.copySign(MAX_VOLTAGE.in(Volts), voltage.in(Volts)));
+    }
+    return voltage;
+  }
+
+  /** Converts linear displacement of motors to angular distances of drivetrain */
+  public static Measure<Angle> distanceToAngle(Measure<Distance> distance) {
```

Comment:
Yeah these util methods should either be moved into DiffDriveIO, its implementations, SparkUtils, or a separate utils file, or something else.

If they're only used in singular places, you may as well just place this code there as well.